### PR TITLE
Vladbez montreal

### LINKED ISSUE #7: Create a new file for vladbez Montreal
- [ ] Add commits
- [ ] Create a branch
- [ ] Open a PR
- [ ] Discuss
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by vladbez**
closes #7

### CODE REVIEW

File: vladbez-Montreal.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Montreal
+
+### Recommendations
+- Eat at Pizza Pizza!
```
Comment (kfonjungo) [inline vladbez-Montreal.md:4]: Which toppings will you recommend.

**Review by kfonjungo — COMMENTED** [review]
This looks good.

**Review by kfonjungo — APPROVED** [review]

**Review by rdsim8589 — APPROVED** [review]
good formatting and a good places to eat

**Review by sandeepbl — APPROVED** [review]
Awesome :+1:

**Pull request merged**